Patch104pZH/GameFilesEdited/Data/INI/FXList.ini

```
@@ -2750,6 +2750,38 @@ FXList FX_OverlordExplosionOneFinal
   End
 End
 
+; -----------------------------------------------------------------------------------
+; Patch104p, A copy of FX_OverlordExplosionOneFinal, suicide variant
+; A death weapon spawns the scorch.
+; -----------------------------------------------------------------------------------
+FXList FX_Suicide_OverlordExplosionOneFinal
```

```suggestion
FXList Nuke_FX_OverlordExplosionOneFinal
```
so: 

- both appear when grepping `FX_OverlordExplosionOneFinal`
- it's obvious that this is a Nukegen exclusive variant (name convention).

Why not make this effect spawn the (presumably bigger) scorch mark? Meaning, remove the scorch mark from the death weapon and keep that responsibility on SlowDeathBehavior for all vehicles (consistency).

Is the before Isotope upgrade scorch mark bigger?

---

Yes we can change name.

I will need to check about spawning scorch via SlowDeath effect instead of death weapon. As for other units with death weapons, I also opted to spawn scorch with death weapon. I think reason being, that death weapon can eventually be delayed, so if scorch is tied to death weapon effects, then there is strong guarantee that it has proper timing.

---

I checked reason for scorchless slow death effect + weapon scorch vs the other way around.

There are 2 reasons:

1. `SlowDeathBehavior` can have `DestructionDelay`, but `FireWeaponWhenDeadBehavior` not.
2. An object can have different death weapons depending on upgrades, for example Bomb Truck.

For these reasons the death weapon bound scorch is good setup. I have done it this way for all the scorch changes and fixes.

---

Effect renamed to `ScorchlessFX_OverlordExplosionOneFinal`